README.md

```diff
@@ -381,7 +382,12 @@ context('Cookie', () => {
   it('user should be logged in still', () => {
     cy.visit('/');
 
-    cy.get('[data-test="user-email"]').should(e =>
+    // or if you use @testing-library/cypress
+    // cy.findByTestId('user-email').should(
```

Comment:
I see other tests in this PR using `cy.findByTestId()`. For consistency and reliability, should we use `cy.findByTestId()` here?

Reply:
Oh never mind this is the README example 😂